src/libraries/System.Security.Cryptography.Pkcs/tests/SignedCms/SignedCmsTests.netcoreapp.cs

```diff
@@ -126,7 +126,7 @@ public static void CounterSignCmsUsingExplicitDSAKeyForFirstSignerAndECDsaForCou
             }
         }
 
-        [Fact]
+        [ConditionalFact(typeof(SignatureSupport), nameof(SignatureSupport.SupportsRsaSha1Signatures))]
```

Comment:
Same feedback as for the RSA one:

`VerifyCounterSignatureWithExplicitPrivateKey` could be changed to use SHA256, except in the case of DSA. You would just need to do this for both the signer and the counter signer.

That also addresses my other next potential feedback, which is that we are conditioning this test based on RSA+SHA1 signing capabilities, when really the test is exercising ECDsa. But if we just "make it work" then we don't need to be conditional at all, except in the case of DSA.